modules/virt-editing-vmis-web.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/cnv-manage-vmis.adoc
+
+[id="virt-editing-vmis-web_{context}"]
+= Editing a standalone virtual machine instance using the web console
+
+You can edit annotations and labels for a standalone virtual machine instance (VMI) using the web console. Other items displayed in the *Details* page for a standalone VMI are not editable.
+
+.Procedure
+
+. Click *Workloads* -> *Virtualization* from the side menu. A list of virtual machines (VMs) and standalone VMIs display.
+
+. Select a standalone VMI to open the *Virtual Machine Instance Overview* screen.
+
+. Click the *Details* tab.
+
+. Click the pencil icon that is located on the right side of *Annotations*.
+
+. Make the relevant changes and click *Save.*.
+
+[NOTE]
```

Comment:
This is a procedure, so I don't think it should be in a note.

Reply:
Do you want the text in this note to be removed from the note and added as an additional step in this procedure? It was presented as a note because this task is performed from the Actions menu, not within the Details tab. This separate note was acceptable to QE. If this note is removed, this information does not fit into the procedure above which is performed in the Details tab. The Action menu is available in multiple tabs, and only for this release this exception is being made. In the next release, this task is being moved into the Details tab and can be done by clicking on a pencil.
Where would you move this note?

Reply:
I am not seeing procedures in our cnv doc for an item that is only selected from the Actions menu. Do you want a separate module created that shows the user how to click the Actions menu and then select "Edit Labels"?

Reply:
In general, we shouldn't put multi-step processes (click actions, click edit labels, make changes, click save) in notes. The module starts with "you can edit annotations and labels" but then the part of the module that discusses labels is just a note, which makes the module seem imbalanced. I think you could add a step in the middle of the procedure that branches off like this:

. Edit the VMI.
.. To edit annotations:
... Click the pencil icon that is located on the right side of *Annotations*.
.. To edit labels:
... Click *Actions*.
... Select *Edit Labels*.

Then you'd keep your final step in place.

Reply:
Matt Carleton is working on a design spec where Labels will be editable in 2.5 release for VMIs.

Reply:
The guidance from QE was that a task performed in the Actions menu should not belong to the procedure that talks about how a task is performed  in the Details page. Which is why I added a note after the procedure to delineate that this task is performed from the Actions menu. CCS thought process differs from QE's thought process in this scenario!

Reply:
I am going to write this up in a Google doc and share with you first before finalizing the content for the PR.

Reply:
Based on our Slack chat, I am leaving the Note for Edit Labels as it is in this module. Again, this note only applies to the 2.4 release. For the 2.5 release, the Edit Labels feature will get the same treatment as the Edit Annotations feature and the module structure and the assembly will be revamped. So this treatment of Edit Labels applies only to the 2.4 release. Thanks for your time and the discussion!